.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. If this is your first time, please read our contributor guidelines: https://github.com/vmware-tanzu/carvel-kapp-controller/blob/develop/CONTRIBUTING.md and developer guide https://github.com/vmware-tanzu/carvel-kapp-controller/blob/develop/docs/dev.md
+-->
+
+#### What type of PR is this?
+
+<!--
+Please add a label to your pull request that matches the type of change you are making.
+
+Some Examples include:
+bug
+documentation
+enhancement
+discussion
+question
+-->
+
+#### What this PR does / why we need it:
+
+#### Which issue(s) this PR fixes:
+<!--
+If no issue exists for this change, please create an issue and link it here.
+-->
+Fixes #
+
+#### Does this PR introduce a user-facing change?
+<!--
+If no, just write "NONE" in the release-note block below.
+If yes, a release note is required:
+Enter your extended release note in the block below. 
+
+-->
+```release-note
+
+```
```

Comment:
Can we make use of this? I guess I worry about having this now as there is no guarantee we will go through each pull request to get this release note. If there is some goal of including automation, I think that makes sense.